File: One Task/Task 1/c programming/41.c
// Matrix Subtraction

#include <stdio.h>

int main() {

    int m1[3][3] = {
        {1, 2, 2},
        {3, 0, 4},
        {4, 1, 2}
    };

    int m2[3][3] = {
        {3, 4, 1},
        {4, 2, 2},
        {3, 2, 1}
    };

    int ans[3][3];

    for(int i=0; i<3; i++) {
        for(int j=0; j<3; j++) {
            ans[i][j] = m1[i][j] - m2[i][j];
            printf("%2d ", ans[i][j]);
        }
        printf("\n");
    }

    return 0;
}

File: Without Node Module/Xampp/htdocs/core/week-1/c-programs/programs/41.php
<?php
    $matrix1 = [[1, 2, 3], [4, 5, 6], [7, 8, 9]];
    $matrix2 = [[9, 8, 7], [6, 5, 4], [3, 2, 1]];
    $resultMatrix = [];

    for ($i = 0; $i < count($matrix1); $i++) {
        for ($j = 0; $j < count($matrix1[$i]); $j++) {
            $resultMatrix[$i][$j] = $matrix1[$i][$j] - $matrix2[$i][$j];
        }
    }

    echo "Matrix Subtraction:<br>";
    printMatrix($resultMatrix);

    function printMatrix($matrix) {
        foreach ($matrix as $row) {
            echo implode(", ", $row) . "<br>";
        }
    }
?>
